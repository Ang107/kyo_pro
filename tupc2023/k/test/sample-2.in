6
2 3 4
13 11 0
3 8 5
20 24 39
21 3 32
3 13 0
