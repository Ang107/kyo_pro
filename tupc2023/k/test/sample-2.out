Yes
2 5 4
6 1 3
No
Yes
1 19 13 18 14 21 17 22
5 9 10 2 20 16 15 23
7 12 11 4 3 8 24 6
No
No
Yes
1 2 3 6 7 9 11 13 14 17 18 19 21
5 8 15 16 25 24 35 32 10 4 20 12 30
22 23 26 27 28 29 31 33 34 36 37 38 39
